ui/app/AppLayouts/Profile/stores/WalletStore.qml

```diff
@@ -11,6 +11,7 @@ QtObject {
     property var collectibles: root.walletModule.collectiblesModel
 
     property var accountSensitiveSettings: Global.appIsReady? localAccountSensitiveSettings : null
+    property var dappList: Global.appIsReady? dappPermissionsModule.dapps : null
```

Comment:
Do you use it somewhere?

Reply:
Not directly here, was just giving warnings somewhere